### PR TITLE
leading and trailing space validation

### PR DESCRIPTION
**Pull request by schwankl**

### CODE REVIEW

**Comment by oneops-admin** [comment]
Build 13

File: lib/base/globalvar/metadata.rb [hunk 1]
```diff
@@ -26,6 +26,7 @@
           :default     => '',
           :format      => {
             :category => 'Value',
+            :pattern  => '[^ ]+.*[^ ]',
```
Comment (sureshg) [inline lib/base/globalvar/metadata.rb]: This won't match other whitespace characters `[\r\n\t\f ]` specially tab space. Use the whitespace literal `\s`.e.g:  `[^\s]+.*[^\s]`

**Comment by oneops-admin** [comment]
Build 14

**Comment by oneops-admin** [comment]
Build 15

**Comment by oneops-admin** [comment]
Build 16

**Comment by oneops-admin** [comment]
Build 17

**Comment by oneops-admin** [comment]
Build 18

**Comment by oneops-admin** [comment]
Build 19

**Comment by oneops-admin** [comment]
Build 20

**Comment by oneops-admin** [comment]
Build 21

**Comment by oneops-admin** [comment]
Build 22

**Comment by oneops-admin** [comment]
Build 23

**Pull request merged**